packages/playground/website/public/logger.php

```diff
@@ -66,12 +70,14 @@ function validate_message($message)
 if (!isset($_POST['message'])) {
     response(false, 'No message provided');
 }
-$text = $_POST['message'];
+$message = urldecode($_POST['message']);
+$message = str_replace("\r\n", "\n", $message); // Replace \r\n with \n
```

Comment:
Why?

Reply:
This broke validation checks when I submitted a report from Linux. It will be removed when I replace message with multiple POST fields.

Reply:
Removed in [74f6c7a](https://github.com/WordPress/wordpress-playground/pull/1102/commits/74f6c7a6907d7211e3c5cbacf933ec2d32dc5c4a)